andela/travel_tool_front
# #160848557 Modify delete comment dialog

#### What does this PR do?
- Remove scrollbar from modal
- Change mouse pointer on hover of delete button to pointer
- Update text colors, margins and paddings to match mockup design

#### Description of Task to be completed?
- Update styles to reflect dialog on [design mockup](https://projects.invisionapp.com/d/main#/console/14861108/320832646/preview)

#### How should this be manually tested?
- Clone this repo - git clone https://github.com/andela/travel_tool_front.git
- Switch to the repo directory - cd travel_tool_front
- Check out this branch - git checkout ft-rejected-request-notification-160833565
- Install dependencies and also start the server - make start
- Create a new travel request
- Click on a request and add a comment
- Click the delete button

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160848557](https://www.pivotaltracker.com/story/show/160848557)

#### Screenshots (if appropriate)
<img width="901" alt="screen shot 2018-09-28 at 5 07 38 pm" src="https://user-images.githubusercontent.com/23608060/46220089-0c36ab00-c341-11e8-9269-b154d636f2c2.png">

#### Questions:
